simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -2,7 +2,7 @@
   <param name="simulate" value="true" />
```

Comment:
Hey I noticed I made a goof - this needs to be:
```<param name="/use_sim_time" value="true" />```
I think something used to set that but it has gotten removed.